modules/RTC/TPCUtils.js

```diff
@@ -38,44 +38,52 @@ export class TPCUtils {
         const standardBitrates = {
             low: LD_BITRATE,
             standard: SD_BITRATE,
-            high: HD_BITRATE
+            high: HD_BITRATE,
+            ssHigh: HD_BITRATE
         };
 
         // Check if the max. bitrates for video are specified through config.js videoQuality settings.
         // Right now only VP8 bitrates are configured on the simulcast encodings, VP9 bitrates have to be
         // configured on the SDP using b:AS line.
         this.videoBitrates = bitrateSettings ?? standardBitrates;
-        const encodingBitrates = this.videoBitrates.VP8 ?? this.videoBitrates;
-
-        /**
-         * The startup configuration for the stream encodings that are applicable to
-         * the video stream when a new sender is created on the peerconnection. The initial
-         * config takes into account the differences in browser's simulcast implementation.
-         *
-         * Encoding parameters:
-         * active - determine the on/off state of a particular encoding.
-         * maxBitrate - max. bitrate value to be applied to that particular encoding
-         *  based on the encoding's resolution and config.js videoQuality settings if applicable.
-         * rid - Rtp Stream ID that is configured for a particular simulcast stream.
-         * scaleResolutionDownBy - the factor by which the encoding is scaled down from the
-         *  original resolution of the captured video.
-         */
-        this.localStreamEncodingsConfig = [
+        this.encodingBitrates = this.videoBitrates.VP8 ?? this.videoBitrates;
+    }
+
+    /**
+     * The startup configuration for the stream encodings that are applicable to
+     * the video stream when a new sender is created on the peerconnection. The initial
+     * config takes into account the differences in browser's simulcast implementation.
+     *
+     * Encoding parameters:
+     * active - determine the on/off state of a particular encoding.
+     * maxBitrate - max. bitrate value to be applied to that particular encoding
+     *  based on the encoding's resolution and config.js videoQuality settings if applicable.
+     * rid - Rtp Stream ID that is configured for a particular simulcast stream.
+     * scaleResolutionDownBy - the factor by which the encoding is scaled down from the
+     *  original resolution of the captured video.
+     *
+     *  @param {VideoType} videoType
+     */
+    _getVideoStreamEncodings(videoType) {
+        const maxVideoRate = videoType === VideoType.DESKTOP
+            ? this.encodingBitrates.ssHigh : this.encodingBitrates.high;
```

Comment:
For VP8 case, when `videoQuality.maxBitratesVideo.VP8.ssHigh` is not defined in config.js, `maxVideoRate` will be `undefined`. Can we change this to

```
maxVideoRate = videoType === VideoType.DESKTOP && this.encodingBitrates.ssHigh
    ? this.encodingBitrates.ssHigh
    : this.encodingBitrates.high;
```